src/Client/components/ques/ProblemSolveQues.tsx

```
@@ -50,14 +58,36 @@ const ProblemItem: React.FC<ProblemItemProps> = ({
             />
         ))}
         <div className="w-full flex justify-between">
-            <div>
+            <div className="flex gap-3">
                 <Input
                     className="w-32"
                     type="number"
                     label="Points"
                     value={problem.points.toString()}
                     onChange={(e) => onPointsChange(parseInt(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)